modules/nextflow/src/main/groovy/nextflow/k8s/client/ClientConfig.groovy

```diff
@@ -37,6 +37,8 @@ class ClientConfig {
 
     String namespace
 
+    Integer maxErrorRetry = 5
```

Comment:
Checking better think this is useless. The client config is used to discover the K8s config in the HOME/.kube/config path. 

Instead this should go in the [K8sConfig](https://github.com/nextflow-io/nextflow/blob/979e39fd0a5e87b1d57918a424382e35d2af1951/modules/nextflow/src/main/groovy/nextflow/k8s/K8sConfig.groovy#L75-L75), tho is not easy then to access it from the client